applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -32,15 +32,14 @@
 namespace Kratos
 {
 
-template <unsigned int TDim, unsigned int TNumNodes>
 class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
 {
 public:
     using SizeType = std::size_t;
 
     KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(UPwBaseElement);
 
-    explicit UPwBaseElement(IndexType NewId = 0) : Element(NewId) {}
+    using Element::Element;
```

Comment:
I don't understand this replacement. Could you please explain?

Reply:
It was a code smell and SonarCloud pointed that exactly the same statement in `Element `and suggested to use `using`. Frankly speaking, I do not fully understand how it works.